drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -167,15 +165,15 @@ def items_without_answers():
             "display_zone_labels": self.data.get('displayLabels', False),
             "display_zone_borders": self.data.get('displayBorders', False),
             "items": items_without_answers(),
-            "title": self.display_name,
+            "title": self.ugettext(self.display_name),
             "show_title": self.show_title,
             "problem_text": self.question_text,
             "show_problem_header": self.show_question_header,
             "target_img_expanded_url": self.target_img_expanded_url,
-            "target_img_description": self.target_img_description,
-            "item_background_color": self.item_background_color or None,
-            "item_text_color": self.item_text_color or None,
-            "initial_feedback": self.data['feedback']['start'],
+            "target_img_description": self.ugettext(self.target_img_description),
+            "item_background_color": self.ugettext(self.item_background_color) or None,
+            "item_text_color": self.ugettext(self.item_text_color) or None,
```

Comment:
@mattdrayer `item_background_color` and `item_text_color` should not be translated: They are used to set CSS properties of DOM elements representing items, so translating them would break item styles.